numpy/core/src/umath/loops_unary_fp.dispatch.c.src

```diff
@@ -0,0 +1,219 @@
+/*@targets
+ ** $maxopt baseline
+ ** sse2 vsx2 neon
+ **/
+/**
+ * Force use SSE only on x86, even if AVX2 or AVX512F are enabled
+ * through the baseline, since scatter(AVX512F) and gather very costly
+ * to handle non-contiguous memory access comparing with SSE for
+ * such small operations that this file covers.
+*/
+#define NPY_SIMD_FORCE_128
```

Comment:
@r-devulap, I enabled `SSE` and dropped `AVX2` and `AVX512F` since there's no performance gain for contiguous arrays,
also the emulated version of partial and non-contiguous memory load/store intrinsics are faster than gather and scatter(AVX512F) when I unroll by x4/x2.